LPS-201903 DSM filter nested fields

boton opened the pull request:
### Issue 
- [LPS-201903](https://liferay.atlassian.net/browse/LPS-201903) Allow users to search for fields in a hierarchical list 
- under FF `feature.flag.LPS-186871=true`

### Search UI

The main concept is to provide a filtering option for hierarchical fields on DSM. Currently, the tree automatically expands the path of results to display the match.

https://github.com/liferay-frontend/liferay-portal/assets/67901/b0e98b7d-ba0b-4bee-86b5-239322e7330c


### Questions

- I've updated the `saveFDSFields` function to save all fields, not just the filtered ones. However, I'm uncertain. @markocikos, could you please review it? 🥺
- I attempted to limit the modal width to size `xl`, but I lost the full height. Any ideas on how to maintain both full height and max-width?

### Still TODO
- Limit the width of the modal
- Highlight the match in the results

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-17.liferay.com/job/test-portal-source-format/6642/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6642/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/977b899d04df122a3f482009ee162404729384cb">977b899d04df122a3f482009ee162404729384cb</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/boton/liferay-portal/tree/LPS-201903-DSM-filter-nested-fields">LPS-201903-DSM-filter-nested-fields</a><br/>Branch GIT ID: <a href="https://github.com/boton/liferay-portal/commit/d57278ae47af7792827c2d275673ea0663fd36eb">d57278ae47af7792827c2d275673ea0663fd36eb</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-17.liferay.com/job/test-portal-source-format/6642/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6642/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-17.liferay.com/job/test-portal-source-format/6642/">test-portal-source-format#6642</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-17.liferay.com/userContent/jobs/test-portal-source-format/builds/6642/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>sf<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3843">liferay-frontend#3843</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2462305696">[master] ci:test:sf - boton &gt; liferay-frontend - PR#3843 - 2023-11-20[09:24:09]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-17.liferay.com/job/publish-testray-report/3622/">publish-testray-report#3622</a><br/></div></div>

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/fields/modal_content/AddFieldsModalContent.tsx:
```diff
@@ -158,49 +214,142 @@ const AddFieldsModalContent = ({
 		});
 	}, [savedFDSFields, fdsView]);
 
+	useEffect(() => {
+		if (initialFields) {
+			const filteredItems = filterItems(initialFields, debouncedSearch);
+			setFields(filteredItems);
+
+			if (debouncedSearch) {
+				const filteredKeys = getItemsKeys(filteredItems);
+				setExpandedKeys(filteredKeys);
+			}
+		}
+	}, [initialFields, debouncedSearch]);
```
Do we need `useEffect` here? It looks to me like the dependencies will never change. If that's the case, we should be able to set initial setting of states in `useState`s.

It's typically wise to avoid `useEffect` whenever possible:
- It adds additional re-render, which is a small hit on performance.
- Additional re-render can lead to bugs and glitchy behavior 
- "event bus" nature of `useEffect` ends up in code that's often hard to debug.

boton replied:
I'm not sure, the `debouncedSearch` is updated when the user type.

markocikos replied:
Oh, my bad! I misread the code, I though `debouncedSearch` was a function. Both `debounce` util in DXP and the old `A.debounce` return a function :sweat_smile: 

But, my point with avoiding `useEffect` is still valid. Is there a way to refactor code to directly modify `fields` and `expandedKeys` states in search input callback?

markocikos commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/fields/modal_content/AddFieldsModalContent.tsx:
```diff
@@ -158,49 +214,142 @@ const AddFieldsModalContent = ({
 			</ClayModal.Header>
 
-			<ClayModal.Body className="bg-light m-4 p-0">
+			<ClayModal.Body className="pt-0 px-0">
```
> I attempted to limit the modal width to size xl, but I lost the full height. Any ideas on how to maintain both full height and max-width?

I kinda like the solution as it is now! Limited internal width in full screen modal seems a nice compromise beween small modal with fixed size and a-bit-too-wide full screen modal. @ugeortiz what do you think?

boton replied:
Sadly on wide screens looks ugly 😓 

![image](https://github.com/liferay-frontend/liferay-portal/assets/67901/d278a245-0525-49f9-922f-17d94c364a78)

markocikos reviewed (commented):
@boton please see comments inline. In addition, take a look at how we're using the `fuzzy` library to **bold** search part of the items text. We're implementing it in several places in DSM. If you prefer, this can be implemented as a separate task.

boton commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/fields/modal_content/AddFieldsModalContent.tsx:
```diff
@@ -5,10 +5,15 @@
+} from '@clayui/management-toolbar';
 import ClayModal from '@clayui/modal';
-import {fetch} from 'frontend-js-web';
+import {useDebounce} from '@clayui/shared';
```
I'm not sure, you are right that it can be simpler. But as a user, I expect the filters to be applied when I have finished typing and not every change. The debounce version feels better than the simple version.

How the frontend filtering is implemented along the DSM?

markocikos replied:
We implemented all filtering without debouncing. See for example in:
- [FDSEntries](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/FDSEntries.tsx#L98-L110)
- [OrderableTable](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/components/OrderableTable.tsx#L311-L331) 
- [ActionsForm](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/actions/ActionForm.tsx#L347-L363)
- [Fields](https://github.com/liferay/liferay-portal/blob/master/modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/fields/Fields.tsx#L146-L161)

But, you may have a point with customers' expectation. We don't _need_ to implement it that way, but we may _want_ to. @ugeortiz What do you think? It is better to wait with search until user stops typing? To me, it seems like immediate results are better, but that might not be ergonomic for a typical user.

boton replied:
Ty! I'll take a look.

Maybe I can remove the debounce for now, and apply it in the future

boton commented:
> @boton please see comments inline. In addition, take a look at how we're using the `fuzzy` library to **bold** search part of the items text. We're implementing it in several places in DSM. If you prefer, this can be implemented as a separate task.

Thanks @markocikos! Nice to know, I'm currently working on it but I make a component for it. I will take a look at `fuzzy` implementation in DSM.

boton commented on modules/apps/frontend-data-set/frontend-data-set-views-web/src/main/resources/META-INF/resources/js/fds_view/fields/modal_content/AddFieldsModalContent.tsx:
```diff
@@ -61,6 +68,48 @@ const applySavedFDSFields = ({
 	return [selectedKeys, fields];
 };
 
+function filterItems(items: Array<IFieldTreeItem>, search: string) {
+	if (!search) {
+		return items;
+	}
+
+	const filteredItems: Array<IFieldTreeItem> = [];
+
+	items.forEach((item) => {
+		const matchesSearch = item.name
+			.toLowerCase()
+			.includes(search.toLowerCase());
+
+		const filteredChildren = item.children?.length
+			? filterItems(item.children, search)
+			: [];
+
+		if (
+			matchesSearch ||
+			(item.children?.length && filteredChildren.length)
+		) {
+			filteredItems.push({
+				...item,
+				children: filteredChildren,
+			});
+		}
+	});
+
+	return filteredItems;
+}
```
Honestly, I don't know how to do it.

We need to render not only if the item matches. The component also has to be rendered if it has matching descendants.

I take advantage of recursion to write the tree from the inside out and only push the path when it is needed.

markocikos replied:
I see, the complexity would need to moved in the rendering part. In the end, we would not gain much, it's quite likely the solution would be even more complex. And we already have this solution that works works perfectly. Good to go! :+1:

boton commented:
I'm working on applying this feedback @markocikos. I will apply your comments and ping you

boton commented:
@markocikos I've applied your feedback, I tried to follow the name conventions and strategies that you suggested to me.

- [x] Debounce removed
- [x] Avoid the useEffect, but I need to add some setFields
- [x] Use regex instead of include/toLowerCase
- [x] naming
  - [x] item[s] => field[s] 
  - [x] search[s] => query[s] 

Instead of moving filterFields to use visit, I added callback support to it 7ccddf2.

I moved to a `draft` because I need to rebase the PR when is reviewed, some commits are adding more confusion than value.

markocikos reviewed (approved):
@boton LGTM! :sparkles: 

We're missing the `fuzzy` bolding, but this can be in a followup task.

boton commented:
Ty @markocikos ❤️ , I'm gonna rebase the PR and `ci:forward` it 🚀 

We have a task for the highlighter here 😄 [`LPS-202271`](https://liferay.atlassian.net/browse/LPS-202271) Highlight the results

boton commented:
ci:forward

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>sf</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-source-format/6118/">test-portal-source-format</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/6118/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:sf - 1 out of 1 jobs passed in 5 minutes</h3><details><summary>Click here for more details.</summary><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a326f83a03da704d0f73d021af1dccf219fc2b63">a326f83a03da704d0f73d021af1dccf219fc2b63</a></p><h4>Sender Branch:</h4><p>Branch Name: <a href="https://github.com/boton/liferay-portal/tree/LPS-201903-DSM-filter-nested-fields">LPS-201903-DSM-filter-nested-fields</a><br/>Branch GIT ID: <a href="https://github.com/boton/liferay-portal/commit/104ea74f7d61687c3147ab1a294d17164c8ce270">104ea74f7d61687c3147ab1a294d17164c8ce270</a></p>1 out of 1jobs PASSED<details><summary><strong>1 Successful Jobs:</strong></summary><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-source-format/6118/">test-portal-source-format</a></li></ul></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-source-format/builds/6118/jenkins-report.html">here</a>.</h5></details></html>

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/10845/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
<html><h3>:heavy_check_mark: ci:test:stable - 31 out of 31 jobs passed</h3><h3>:x: ci:test:relevant - 114 out of 133 jobs passed in 2 hours 3 minutes</h3><details><summary>Click here for more details.</summary><br/><p>This pull is eligible for <a href="https://grow.liferay.com/share/CI+liferay-continuous-integration+GitHub+Commands#General-Commands">reevaluation</a>. When this <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)/192">upstream build</a> has completed, using the following CI command will compare this pull request result against a more recent upstream result:<pre>ci:reevaluate:1277401_10845</pre></p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a><br/>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/a326f83a03da704d0f73d021af1dccf219fc2b63">a326f83a03da704d0f73d021af1dccf219fc2b63</a></p><h4>Upstream Comparison:</h4><p>Branch GIT ID: <a href="https://github.com/liferay/liferay-portal/commit/82d959e5d37f7cf326fb8c17a5a7b7d1a6713ba5">82d959e5d37f7cf326fb8c17a5a7b7d1a6713ba5</a><br/>Jenkins Build URL: <a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2464726595">EE Development Acceptance (master) - 190 - 2023-11-23[16:39:30]</a></p><details><summary><strong>ci:test:stable - 31 out of 31 jobs PASSED</strong></summary><details><summary><strong>31 Successful Jobs:</strong></summary><ul/></details></details><details><summary><strong>ci:test:relevant - 114 out of 133 jobs PASSED</strong></summary><div><h4>19 Failed Jobs:</h4><ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/10845/"><strike>test-portal-acceptance-pullrequest(master)</strike></a></li><li>build-iw-jdk8<ul><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323718/">build-iw-jdk8/0/0</a></li></ul></li><li>central-requirements-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/751476/">central-requirements-jdk8/0/0</a></li></ul></li><li>empty-osgi-core-dir-mysql57-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/464457/">empty-osgi-core-dir-mysql57-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-db2115-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/333388/">functional-smoke-tomcat90-db2115-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-hypersonic20-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354125/">functional-smoke-tomcat90-hypersonic20-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk11_open<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308213/">functional-smoke-tomcat90-mysql57-jdk11_open/0/0</a></li></ul></li><li>functional-smoke-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308216/">functional-smoke-tomcat90-mysql57-jdk8_stable/0/0</a></li></ul></li><li>functional-smoke-tomcat90-oracle193-jdk8<ul><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773957/">functional-smoke-tomcat90-oracle193-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321338/">functional-smoke-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354127/">functional-smoke-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>functional-smoke-tomcat90-sqlserver2019-jdk8<ul><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/824819/">functional-smoke-tomcat90-sqlserver2019-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb104-jdk8<ul><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/769591/">functional-smoke-wildfly180-mariadb104-jdk8/0/0</a></li></ul></li><li>functional-smoke-wildfly180-mariadb106-jdk8<ul><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354124/">functional-smoke-wildfly180-mariadb106-jdk8/0/0</a></li></ul></li><li>functional-tomcat90-mysql57-chrome100-jdk8<ul><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354290/">functional-tomcat90-mysql57-chrome100-jdk8/0/0</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354291/">functional-tomcat90-mysql57-chrome100-jdk8/0/1</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/444318/">functional-tomcat90-mysql57-chrome100-jdk8/0/2</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/824822/">functional-tomcat90-mysql57-chrome100-jdk8/0/3</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/302321/">functional-tomcat90-mysql57-chrome100-jdk8/0/4</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837319/">functional-tomcat90-mysql57-chrome100-jdk8/0/5</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/869208/">functional-tomcat90-mysql57-chrome100-jdk8/0/6</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734970/">functional-tomcat90-mysql57-chrome100-jdk8/0/7</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679082/">functional-tomcat90-mysql57-chrome100-jdk8/0/8</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/271061/">functional-tomcat90-mysql57-chrome100-jdk8/0/9</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/300529/">functional-tomcat90-mysql57-chrome100-jdk8/0/10</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/796273/">functional-tomcat90-mysql57-chrome100-jdk8/0/11</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/331363/">functional-tomcat90-mysql57-chrome100-jdk8/0/12</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/319754/">functional-tomcat90-mysql57-chrome100-jdk8/0/13</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317385/">functional-tomcat90-mysql57-chrome100-jdk8/0/14</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354128/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/15</strike></a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323721/">functional-tomcat90-mysql57-chrome100-jdk8/0/16</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354292/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/17</strike></a></li><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308215/">functional-tomcat90-mysql57-chrome100-jdk8/0/18</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316070/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/19</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942557/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/20</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/787861/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/21</strike></a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/751473/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/22</strike></a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/710798/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/0/23</strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321337/">functional-tomcat90-mysql57-chrome100-jdk8/0/24</a></li><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/319753/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/0</strike></a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679080/">functional-tomcat90-mysql57-chrome100-jdk8/1/1</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317382/">functional-tomcat90-mysql57-chrome100-jdk8/1/2</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317383/">functional-tomcat90-mysql57-chrome100-jdk8/1/3</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942555/">functional-tomcat90-mysql57-chrome100-jdk8/1/4</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/444315/">functional-tomcat90-mysql57-chrome100-jdk8/1/5</a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/271058/">functional-tomcat90-mysql57-chrome100-jdk8/1/6</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/464454/">functional-tomcat90-mysql57-chrome100-jdk8/1/7</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316068/">functional-tomcat90-mysql57-chrome100-jdk8/1/8</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679079/">functional-tomcat90-mysql57-chrome100-jdk8/1/9</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837317/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/10</strike></a></li><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/271059/">functional-tomcat90-mysql57-chrome100-jdk8/1/11</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317381/">functional-tomcat90-mysql57-chrome100-jdk8/1/12</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843609/">functional-tomcat90-mysql57-chrome100-jdk8/1/13</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321335/">functional-tomcat90-mysql57-chrome100-jdk8/1/14</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/331361/">functional-tomcat90-mysql57-chrome100-jdk8/1/15</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321336/">functional-tomcat90-mysql57-chrome100-jdk8/1/16</a></li><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/464455/">functional-tomcat90-mysql57-chrome100-jdk8/1/17</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/824820/">functional-tomcat90-mysql57-chrome100-jdk8/1/18</a></li><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837318/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/19</strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354123/">functional-tomcat90-mysql57-chrome100-jdk8/1/20</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/331360/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/1/21</strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321334/">functional-tomcat90-mysql57-chrome100-jdk8/1/22</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679077/">functional-tomcat90-mysql57-chrome100-jdk8/1/23</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316067/">functional-tomcat90-mysql57-chrome100-jdk8/1/24</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/710801/">functional-tomcat90-mysql57-chrome100-jdk8/2/0</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/953642/">functional-tomcat90-mysql57-chrome100-jdk8/2/1</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323723/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/2/2</strike></a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354129/">functional-tomcat90-mysql57-chrome100-jdk8/2/3</a></li><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317386/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/2/4</strike></a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321340/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/2/5</strike></a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/769592/">functional-tomcat90-mysql57-chrome100-jdk8/2/6</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/333391/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/2/7</strike></a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354294/"><strike>functional-tomcat90-mysql57-chrome100-jdk8/2/8</strike></a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/787864/">functional-tomcat90-mysql57-chrome100-jdk8/2/9</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289764/">functional-tomcat90-mysql57-chrome100-jdk8/2/10</a></li><li><a href="https://test-1-14.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/300530/">functional-tomcat90-mysql57-chrome100-jdk8/2/11</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942558/">functional-tomcat90-mysql57-chrome100-jdk8/2/12</a></li><li><a href="https://test-1-34.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/773956/">functional-tomcat90-mysql57-chrome100-jdk8/2/13</a></li><li><a href="https://test-1-17.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/302322/">functional-tomcat90-mysql57-chrome100-jdk8/2/14</a></li><li><a href="https://test-1-35.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/734971/">functional-tomcat90-mysql57-chrome100-jdk8/2/15</a></li><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316071/">functional-tomcat90-mysql57-chrome100-jdk8/2/16</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/444319/">functional-tomcat90-mysql57-chrome100-jdk8/2/17</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/824823/">functional-tomcat90-mysql57-chrome100-jdk8/2/18</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/331364/">functional-tomcat90-mysql57-chrome100-jdk8/2/19</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/796272/">functional-upgrade-tomcat90-mysql57-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843610/">functional-upgrade-tomcat90-mysql57-jdk8/0/1</a></li><li><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354126/">functional-upgrade-tomcat90-mysql57-jdk8/0/2</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679081/">functional-upgrade-tomcat90-mysql57-jdk8/0/3</a></li><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/796271/">functional-upgrade-tomcat90-mysql57-jdk8/0/4</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354289/">functional-upgrade-tomcat90-mysql57-jdk8/0/5</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323720/">functional-upgrade-tomcat90-mysql57-jdk8/0/6</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942556/">functional-upgrade-tomcat90-mysql57-jdk8/0/7</a></li><li><a href="https://test-1-30.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/824821/">functional-upgrade-tomcat90-mysql57-jdk8/0/8</a></li></ul></li><li>functional-upgrade-tomcat90-mysql57-jdk8_stable<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/271060/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289762/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/1</a></li><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/869207/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/2</a></li><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323719/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/3</a></li><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/444316/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/4</a></li><li><a href="https://test-1-4.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/331362/">functional-upgrade-tomcat90-mysql57-jdk8_stable/0/5</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql144-jdk8<ul><li><a href="https://test-1-25.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/444317/">functional-upgrade-tomcat90-postgresql144-jdk8/0/0</a></li></ul></li><li>functional-upgrade-tomcat90-postgresql153-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308214/">functional-upgrade-tomcat90-postgresql153-jdk8/0/0</a></li></ul></li><li>js-unit-jdk8<ul><li><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316069/">js-unit-jdk8/0/0</a></li></ul></li><li>lpkg-controller-mysql57-jdk8<ul><li><a href="https://test-1-5.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/271062/">lpkg-controller-mysql57-jdk8/0/0</a></li></ul></li><li>modules-compile-jdk8<ul><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/333389/">modules-compile-jdk8/0/0</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843611/">modules-compile-jdk8/0/1</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/710799/">modules-compile-jdk8/0/2</a></li><li><a href="https://test-1-37.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/953643/">modules-compile-jdk8/0/3</a></li><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/787863/">modules-compile-jdk8/0/4</a></li><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/751475/">modules-compile-jdk8/0/5</a></li><li><a href="https://test-1-39.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/710800/">modules-compile-jdk8/0/6</a></li><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843612/">modules-compile-jdk8/0/7</a></li><li><a href="https://test-1-19.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/333390/">modules-compile-jdk8/0/8</a></li><li><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354293/">modules-compile-jdk8/0/9</a></li><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679078/">modules-compile-jdk8/0/10</a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942554/">modules-compile-jdk8/0/11</a></li></ul></li><li>modules-integration-mysql57-jdk8<ul><li><a href="https://test-1-32.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/843613/"><strike>modules-integration-mysql57-jdk8/0/0</strike></a></li><li><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942559/">modules-integration-mysql57-jdk8/0/1</a></li></ul></li><li>modules-integration-mysql57-jdk8_stable<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/869209/">modules-integration-mysql57-jdk8_stable/0/0</a></li><li><a href="https://test-1-29.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/769590/">modules-integration-mysql57-jdk8_stable/0/1</a></li></ul></li><li>modules-semantic-versioning-jdk8<ul><li><a href="https://test-1-33.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/679083/">modules-semantic-versioning-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8<ul><li><a href="https://test-1-9.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/323722/">modules-unit-jdk8/0/0</a></li></ul></li><li>modules-unit-jdk8_stable<ul><li><a href="https://test-1-16.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/289763/">modules-unit-jdk8_stable/0/0</a></li></ul></li><li>modules-unit-project-templates-jdk8<ul><li><a href="https://test-1-36.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/787862/">modules-unit-project-templates-jdk8/0/0</a></li><li><a href="https://test-1-6.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/321339/">modules-unit-project-templates-jdk8/0/1</a></li></ul></li><li>patching-tool-mysql57-jdk8<ul><li><a href="https://test-1-28.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/751474/">patching-tool-mysql57-jdk8/0/0</a></li></ul></li><li>pmd-jdk8<ul><li><a href="https://test-1-40.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/464456/">pmd-jdk8/0/0</a></li></ul></li><li>portal-license-smoke-mysql57-jdk8<ul><li><a href="https://test-1-31.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/869210/">portal-license-smoke-mysql57-jdk8/0/0</a></li></ul></li><li>rest-builder-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/796275/">rest-builder-jdk8/0/0</a></li></ul></li><li>semantic-versioning-jdk8<ul><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837320/"><strike>semantic-versioning-jdk8/0/0</strike></a></li></ul></li><li>service-builder-jdk8<ul><li><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/796274/">service-builder-jdk8/0/0</a></li></ul></li><li>unit-jdk8<ul><li><a href="https://test-1-21.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/317384/">unit-jdk8/0/0</a></li></ul></li><li>unit-jdk8_stable<ul><li><a href="https://test-1-2.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/319755/">unit-jdk8_stable/0/0</a></li></ul></li><li>workspaces-compile-jdk8<ul><li><a href="https://test-1-11.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/308212/">workspaces-compile-jdk8/0/0</a></li></ul></li></ul></div><details><summary><strong>114 Successful Jobs:</strong></summary><ul/></details></details><h5>For more details click <a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/jenkins-report.html">here</a>.</h5><hr/><h4>Failures unique to this pull:</h4><ol><li><div><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837320//consoleText">semantic-versioning-jdk8/0/0</a><pre>clean-up-java-processes:
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/clean_up_java_processes.sh
[stopwatch] [run.batch.test.setup: 26:47.501 sec]
     [echo] run.batch.test.action.start.timestamp: 11-24-2023 07:18:47:785 PST
     [echo] Checking for baseline log files.
     [echo] 
     [echo] ##
     [echo] ## /opt/dev/projects/github/liferay-portal/baseline-reports/portal-impl.log
     [echo] ##
     [echo] 
[beanshell] [Baseline Warning] Bundle Version Change Recommended: 95.0.0
[beanshell] 
[stopwatch] [run.batch.test.action: 4.483 sec]
     [echo] Semantic versioning is incorrect.
      [get] Getting: \$\{dependencies.url\}/archive/test-1-27/test-portal-acceptance-pullrequest-downstream(master)/837320consoleText
      [get] To: /opt/dev/projects/github/liferay-portal/20231124071852270.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/20231124071852270.txt
   [delete] Deleting: /opt/dev/projects/github/liferay-portal/null1170652923.properties</pre><ol><li><details><summary><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837320/testReport/com.liferay.jenkins/JenkinsLogAsserterTest">com.liferay.jenkins.JenkinsLogAsserterTest</a></summary><div><ol><li><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/837320//testReport/com.liferay.jenkins/JenkinsLogAsserterTest/testScanJenkinsLog">testScanJenkinsLog</a></li></ol></div></details></li></ol></div></li></ol><hr/><details><summary><strong>Failures in common with <a href="https://test-1-40.liferay.com/job/test-portal-acceptance-upstream-dxp(master)">acceptance upstream results</a> at 82d959e5d37f7cf326fb8c17a5a7b7d1a6713ba5:</strong></summary><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354128//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/15</a><ol><li><div><a href="https://test-1-8.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354128//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DataSet_TheFieldValueIsTranslated_">LocalFile.DataSet#TheFieldValueIsTranslated</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/15/LocalFile.DataSet_TheFieldValueIsTranslated/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/15/LocalFile.DataSet_TheFieldValueIsTranslated/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/15/LocalFile.DataSet_TheFieldValueIsTranslated/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/15/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2464733522/history">Failed 25 of last 25</a></div></li></ol></div></li><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354292//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/17</a><ol><li><div><a href="https://test-1-3.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/354292//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DataSet_TheFieldsAreTranslatedOnPage_">LocalFile.DataSet#TheFieldsAreTranslatedOnPage</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/17/LocalFile.DataSet_TheFieldsAreTranslatedOnPage/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/17/LocalFile.DataSet_TheFieldsAreTranslatedOnPage/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/17/LocalFile.DataSet_TheFieldsAreTranslatedOnPage/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/17/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2464733483/history">Failed 25 of last 25</a></div></li></ol></div></li><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316070//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/19</a><ol><li><div><a href="https://test-1-10.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/316070//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DataSet_AssertTheNewFieldTranslation_">LocalFile.DataSet#AssertTheNewFieldTranslation</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/19/LocalFile.DataSet_AssertTheNewFieldTranslation/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/19/LocalFile.DataSet_AssertTheNewFieldTranslation/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/19/LocalFile.DataSet_AssertTheNewFieldTranslation/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/19/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2464733479/history">Failed 23 of last 25</a></div></li></ol></div></li><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942557//consoleText">functional-tomcat90-mysql57-chrome100-jdk8/0/20</a><ol><li><div><a href="https://test-1-26.liferay.com/job/test-portal-acceptance-pullrequest-downstream(master)/942557//testReport/com.liferay.poshi.runner/PoshiRunner/test_LocalFile_DataSet_CanAssertTheEndpointObjectInDataSetCreated_">LocalFile.DataSet#CanAssertTheEndpointObjectInDataSetCreated</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/20/LocalFile.DataSet_CanAssertTheEndpointObjectInDataSetCreated/index.html.gz">Poshi Report</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/20/LocalFile.DataSet_CanAssertTheEndpointObjectInDataSetCreated/summary.html.gz">Poshi Summary</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/20/LocalFile.DataSet_CanAssertTheEndpointObjectInDataSetCreated/console.txt.gz">Poshi Console</a> - <a href="https://testray.liferay.com/reports/production/logs/2023-11/test-1-27/test-portal-acceptance-pullrequest(master)/10845/functional-tomcat90-mysql57-chrome100-jdk8/0/20/jenkins-console.txt.gz">Console Output</a> - <a href="https://testray.liferay.com/home/-/testray/case_results/2464733413/history">Failed 25 of last 25</a></div></li></ol></div></li><li>...</li></ol></details></details></html><h5>Test bundle downloads:</h5><ul><li><a href="https://files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/liferay-portal-bundle-tomcat.tar.gz">liferay-portal-bundle-tomcat.tar.gz</a> (mirrors: <a href="http://mirrors.dlc.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/liferay-portal-bundle-tomcat.tar.gz">dlc</a>, <a href="http://mirrors.lax.liferay.com/files.liferay.com/private/bundles/test-1-27/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/liferay-portal-bundle-tomcat.tar.gz">lax</a>)</li></ul>

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-27.liferay.com/job/test-portal-acceptance-pullrequest(master)/10845/">test-portal-acceptance-pullrequest(master)#10845</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-27.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/10845/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3843">liferay-frontend#3843</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2464829871">[master] ci:test:relevant - boton &gt; liferay-frontend - PR#3843 - 2023-11-24[06:03:29]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-27.liferay.com/job/publish-testray-report/23383/">publish-testray-report#23383</a><br/></div></div>

liferay-continuous-integration commented:
CI is automatically triggering the following test suites:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

The pull request will automatically be forwarded to the user `brianchandotcom` If the following test suites pass:
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**relevant**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**sf**
- &nbsp;&nbsp;&nbsp;&nbsp;ci:test:**stable**

liferay-continuous-integration commented:
Skipping previously passed test suites:
- `ci:test:sf`
- `ci:test:stable`

liferay-continuous-integration commented:
<h1>Build started.</h1><p>Jenkins is currently running tests in the <code>relevant</code> test suite.</p><h4>Base Branch:</h4><p>Branch Name: <a href="https://github.com/liferay/liferay-portal/tree/master">master</a></p><h4>Job Summary:</h4><p>Job Link: <a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/9800/">test-portal-acceptance-pullrequest(master)</a></p><p><b>For more details click <a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9800/jenkins-report.html">here</a>.</b></p>

liferay-continuous-integration commented:
All required test suite(s) passed.
Forwarding pull request to `brianchandotcom`.
[Console](https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/9800//consoleFull)

liferay-continuous-integration commented:
Pull request has been successfully forwarded to  https://github.com/brianchandotcom/liferay-portal/pull/144445
[Console](https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/9800//consoleFull)

liferay-continuous-integration commented:
<div><div><strong>Jenkins Build:</strong><span></span><a href="https://test-1-38.liferay.com/job/test-portal-acceptance-pullrequest(master)/9800/">test-portal-acceptance-pullrequest(master)#9800</a><br/></div><div><strong>Jenkins Report:</strong><span></span><a href="https://test-1-38.liferay.com/userContent/jobs/test-portal-acceptance-pullrequest(master)/builds/9800/jenkins-report.html">jenkins-report.html</a><br/></div><div><strong>Jenkins Suite:</strong><span></span>relevant<br/></div><div><strong>Pull Request:</strong><span></span><a href="https://github.com/liferay-frontend/liferay-portal/pull/3843">liferay-frontend#3843</a><br/></div><div><strong>Testray Routine:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/builds?testrayRoutineId=1306753894">EE Pull Request</a><br/></div><div><strong>Testray Build:</strong><span></span><a href="https://testray.liferay.com/home/-/testray/runs?testrayBuildId=2465559809">[master] ci:test:relevant - boton &gt; liferay-frontend - PR#3843 - 2023-11-27[06:07:11]</a><br/></div><div><strong>Testray Importer:</strong><span></span><a href="https://test-1-38.liferay.com/job/publish-testray-report/19991/">publish-testray-report#19991</a><br/></div></div>
